BasicAlgorithms/BubbleSort.cs

```diff
@@ -5,20 +5,34 @@ namespace BasicAlgorithms
 {
     public class BubbleSort
     {
-        public static void Sort(int[] Items)
+        public static void Sort(int[] Items, bool userPref = true)
         {
             int i = 0;
             int j = 0;
             for(i= 0; i <= Items.Length - 1; i++)
             {
                 for (j = i + 1; j <= Items.Length - 1; j++)
                 {
-                    if(Items[i] > Items[j])
+                    if(userPref == true)
                     {
-                        int aux = Items[i];
-                        Items[i] = Items[j];
-                        Items[j] = aux;
+                        if (Items[i] > Items[j])
+                        {
+                            int aux = Items[i];
+                            Items[i] = Items[j];
+                            Items[j] = aux;
+                        }
                     }
+                    else
+                    {
+                        if (Items[i] < Items[j])
+                        {
+                            int aux = Items[i];
+                            Items[i] = Items[j];
+                            Items[j] = aux;
+                        }
```

Comment:
Duplicated code inside the if and else branches. Try to extract the logic of comparison into a separate method.